examples/gettingStarted/scheduledMessages.ts

```diff
@@ -0,0 +1,80 @@
+import { Namespace } from "../../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const connectionString = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const queueName = process.env.QUEUE_NAME || "";
+const topicName = process.env.TOPIC_NAME || "";
+const subscriptionName = process.env.SUBSCRIPTION_NAME || "";
+const NUM_OF_MESSAGES = 10;
+
+console.log("Connection string value: ", connectionString);
+console.log("Queue name: ", queueName);
+console.log("Topic name: ", topicName);
+console.log("Subscription name: ", subscriptionName);
+
+let ns: Namespace;
+
+/*
+  This sample demonstrates how .scheduleMessage() API can be used to schedule messages to appear
+  on a Service Bus entity at a specified later time.
+*/
+async function main(): Promise<void> {
+  ns = Namespace.createFromConnectionString(connectionString);
+  try {
+    // schedule messages to appear on queue/topic at a later time.
+    await sendScheduledMessages();
+
+    // retrieve all the messages that were sent to the queue
+    await receiveMessages();
+  } finally {
+    await ns.close();
+  }
+}
+
+async function sendScheduledMessages(): Promise<void> {
+  const client = ns.createQueueClient(queueName); // Use this API to send to a queue
+  // const client = ns.createTopicClient(topicName); // Use this API to send to a topic
+
+  const data = [
+    { name: "Einstein", firstName: "Albert" },
+    { name: "Heisenberg", firstName: "Werner" },
+    { name: "Curie", firstName: "Marie" },
+    { name: "Hawking", firstName: "Steven" },
+    { name: "Newton", firstName: "Isaac" },
+    { name: "Bohr", firstName: "Niels" },
+    { name: "Faraday", firstName: "Michael" },
+    { name: "Galilei", firstName: "Galileo" },
+    { name: "Kepler", firstName: "Johannes" },
+    { name: "Kopernikus", firstName: "Nikolaus" }
+  ];
+
+  for (let index = 0; index < NUM_OF_MESSAGES; index++) {
+    const element = data[index];
```

Comment:
s/element/scientist, maybe? Element is a bit overloaded in scientific contexts :)